src/Concordium/Client/Output.hs

```diff
@@ -921,7 +921,7 @@ showEvent verbose ciM = \case
                     bsl = BSL.fromStrict $ BSS.fromShort bss
 
                     -- First decoding attempt using the token event decoder for the add/remove to/from allow/deny list events.
-                    decodedTokenEvent = Cbor.decodeTokenEvent (Cbor.EncodedTokenEvent (Types.tokenEventTypeBytes etmeType) (Just bss))
+                    decodedTokenEvent = Cbor.decodeTokenEvent (Cbor.EncodedTokenEvent etmeType etmeDetails)
```

Comment:
please update the changelog.

Reply:
In general, the changelog should be updated (not particularly related to the above line). This line was just the first line in the review where I could add a comment.

E.g. aligning the `node` release and the `concordium-client` release to ensure that both have the same feature set when we make the cut.

Reply:
I think we should only add functionally important changes to the changelog. The changelog should be a place to easily figure out, how the software changed from release to release. If you just mirror the git history, you'll end up with a hard to read and redundant document. I don't think this change falls into the functionally important category.

Reply:
If a transaction costs more or less in a new release, it's important to be aware of. However, as it is only an alpha release, feel free to proceed without a changelog entry as we are internally aware of the change and can align the `node`/`concordium-client` release.